README.md

```diff
@@ -1 +1,63 @@
-# angular-what-is-a-controller-readme
\ No newline at end of file
+# What is a Controller?
+
+## Overview
+
+We've talked about controllers before, and now is the time to go into them. Controllers make up a lot of the code we write in Angular so it's important that we learn how to use them correctly and efficiently!
+
+## Objectives
+
+- Describe the role of a Controller
+- Create a Controller
+- Assign the Controller to the module
+- Create an element for Controller scope
+
+## Controllers
+
+You might have noticed that we've mentioned the concept of Controllers quite a lot. These are the backbone for your Angular applications, providing the link between the view and the model.
```

Comment:
In the intro, provide a little bit more context around the responsibility of controllers before discussing the difference between fat and skinny controllers.